.env.example

```diff
@@ -5,6 +5,12 @@ ROLLBAR_ACCESS_TOKEN=
 ROLLBAR_ENV=development
 WSGI_AUTH_EXCLUDE_PATHS=/check
 XSLT_IMAGE_LOCATION=
+DJANGO_DEBUG=True
+# These are deprecated and replaced with ASSETS_ENDPOINT
 PUBLIC_ASSET_BUCKET=tna-caselaw-assets-staging
 S3_REGION=eu-west-2
-DJANGO_DEBUG=True
+# note: this is the *live* assets endpoint -- staging does not
+# typically have the DOCX or PDF available in S3.
+ASSETS_CDN_BASE_URL=https://assets.caselaw.nationalarchives.gov.uk
+# but you would use this staging URL if you wanted it:
+# ASSETS_CDN_BASE_URL=https://tna-caselaw-assets-staging.s3.eu-west-2.amazonaws.com
```

Comment:
I'm broadly documenting "this is the value this would have if you wanted it to point at staging"; setting it to production does make a weird hybrid, but one which I think makes more sense; we could ask people to leave it blank if they wanted that, but that's rather unintuitive too...